services/sidecar/src/simcore_service_sidecar/cli.py

```diff
@@ -34,33 +33,24 @@ def main(
 async def run_sidecar(
     job_id: str, user_id: str, project_id: str, node_id: Optional[str]
 ) -> Tuple[Optional[List[str]], Optional[str]]:
-    try:
-        log.info(
-            "STARTING task %s processing for user %s, project %s, node %s",
-            job_id,
-            user_id,
-            project_id,
-            node_id,
-        )
-        async with DBContextManager() as db_engine:
-            async with RabbitMQ(config=RABBIT_CONFIG) as rabbit_mq:
-                next_task_nodes: Optional[List[str]] = await inspect(
-                    db_engine, rabbit_mq, job_id, user_id, project_id, node_id=node_id
-                )
-                log.info(
-                    "COMPLETED task %s processing for user %s, project %s, node %s",
-                    job_id,
-                    user_id,
-                    project_id,
-                    node_id,
-                )
-                return next_task_nodes, None
-    except asyncio.CancelledError as e:
-        log.warning(
-            "Run cancelled", exc_info=True, stack_info=SIDECAR_LOGLEVEL == logging.DEBUG
-        )
-        return None, f"Run cancelled: {str(e)}"
-    except Exception as e:  # pylint: disable=broad-except
-        error_message = f"run_sidecar caught the following exception: {str(e)}"
-        log.exception(error_message)
-        return None, error_message
+
+    log.info(
+        "STARTING task %s processing for user %s, project %s, node %s",
+        job_id,
+        user_id,
+        project_id,
+        node_id,
+    )
+    async with DBContextManager() as db_engine:
+        async with RabbitMQ(config=RABBIT_CONFIG) as rabbit_mq:
+            next_task_nodes: Optional[List[str]] = await inspect(
+                db_engine, rabbit_mq, job_id, user_id, project_id, node_id=node_id
+            )
+            log.info(
+                "COMPLETED task %s processing for user %s, project %s, node %s",
+                job_id,
+                user_id,
+                project_id,
+                node_id,
+            )
+            return next_task_nodes, None
```

Comment:
If no errors are returned this can be changed to just return next_task_nodes

Reply:
true, done